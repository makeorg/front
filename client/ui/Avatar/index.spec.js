import React from 'react';
import renderer from 'react-test-renderer';
import { Avatar } from './index';

jest.mock('Client/ui/Svg', () => ({ Svg: 'Svg' }));

describe('Avatar', () => {
  it('snapshot by default', () => {
    const component = renderer.create(<Avatar />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
