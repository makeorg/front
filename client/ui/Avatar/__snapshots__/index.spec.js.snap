// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar must match the diff snapshot between default vs custom avatar 1`] = `
Snapshot Diff:
- First value
+ Second value

  <span
    className="sc-1nws3fm-0 kmyGbB"
  >
    <SvgEmptyAvatar
      aria-hidden={true}
-     height={30}
-     width={30}
+     height={200}
+     width={200}
    />
  </span>
`;

exports[`Avatar snapshot by default 1`] = `
.c0 {
  position: static;
  margin-right: 10px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

<span
  className="c0"
>
  <SvgEmptyAvatar
    aria-hidden={true}
    height={30}
    width={30}
  />
</span>
`;
